Sources/Clock2.swift

```diff
@@ -19,6 +19,10 @@ import Foundation
 /// // (... later on ...)
 /// print(Clock.now)
 /// ```
+/// - Note:
+///     Not thread-safe. Use only from single thread.
+///     Caller is responsible to make sure that all features to be called from single thread only.
```

Comment:
This comment is incorrect until we remove (or prove) all calls to POSIX/Darwin system calls.

Reply:
Update the comment to reflect the situation precisely.

Reply:
Understood, I have updated the code accordingly

Reply:
I have removed the note as the current comment is quite sufficient to explain the situation